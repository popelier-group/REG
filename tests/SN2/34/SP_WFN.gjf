# b3lyp/6-311++g(d,p) output=wfn

Single Point Energy

-1 1
 C                 -1.56174600    0.00024000   -0.00077000
 H                 -1.22867600   -0.87389800   -0.54642400
 H                 -1.22899000    0.90984600   -0.48521000
 H                 -1.22821100   -0.03504600    1.02893400
 Br                 1.79744900   -0.00002300    0.00020100
 Cl                -3.41435300   -0.00005700   -0.00011300

frame.wfn
