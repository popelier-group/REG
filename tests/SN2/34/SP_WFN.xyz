
6

C        0.093594    0.346574    1.381613
H        0.818379    0.941110    0.839390
H        0.311385   -0.710165    1.288543
H       -0.913718    0.565590    1.049748
Br       -0.125994   -0.469448   -1.869553
Cl        0.213657    0.797335    3.174530
