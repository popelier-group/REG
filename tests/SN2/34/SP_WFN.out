 Entering Gaussian System, Link 0=/opt/apps/apps/binapps/gaussian/g16c01_em64t_sandybridge/g16/g16
 Initial command:
 /opt/apps/apps/binapps/gaussian/g16c01_em64t_sandybridge/g16/l1.exe "/scratch/v69787ff/gau_temp_1513317.34/Gau-22884.inp" -scrdir="/scratch/v69787ff/gau_temp_1513317.34/"
 Default is to use a total of   2 processors:
                                2 via shared-memory
                                1 via Linda
 Entering Link 1 = /opt/apps/apps/binapps/gaussian/g16c01_em64t_sandybridge/g16/l1.exe PID=     22885.
  
 Copyright (c) 1988-2019, Gaussian, Inc.  All Rights Reserved.
  
 This is part of the Gaussian(R) 16 program.  It is based on
 the Gaussian(R) 09 system (copyright 2009, Gaussian, Inc.),
 the Gaussian(R) 03 system (copyright 2003, Gaussian, Inc.),
 the Gaussian(R) 98 system (copyright 1998, Gaussian, Inc.),
 the Gaussian(R) 94 system (copyright 1995, Gaussian, Inc.),
 the Gaussian 92(TM) system (copyright 1992, Gaussian, Inc.),
 the Gaussian 90(TM) system (copyright 1990, Gaussian, Inc.),
 the Gaussian 88(TM) system (copyright 1988, Gaussian, Inc.),
 the Gaussian 86(TM) system (copyright 1986, Carnegie Mellon
 University), and the Gaussian 82(TM) system (copyright 1983,
 Carnegie Mellon University). Gaussian is a federally registered
 trademark of Gaussian, Inc.
  
 This software contains proprietary and confidential information,
 including trade secrets, belonging to Gaussian, Inc.
  
 This software is provided under written license and may be
 used, copied, transmitted, or stored only in accord with that
 written license.
  
 The following legend is applicable only to US Government
 contracts under FAR:
  
                    RESTRICTED RIGHTS LEGEND
  
 Use, reproduction and disclosure by the US Government is
 subject to restrictions as set forth in subparagraphs (a)
 and (c) of the Commercial Computer Software - Restricted
 Rights clause in FAR 52.227-19.
  
 Gaussian, Inc.
 340 Quinnipiac St., Bldg. 40, Wallingford CT 06492
  
  
 ---------------------------------------------------------------
 Warning -- This program may not be used in any manner that
 competes with the business of Gaussian, Inc. or will provide
 assistance to any competitor of Gaussian, Inc.  The licensee
 of this program is prohibited from giving any competitor of
 Gaussian, Inc. access to this program.  By using this program,
 the user acknowledges that Gaussian, Inc. is engaged in the
 business of creating and licensing software in the field of
 computational chemistry and represents and warrants to the
 licensee that it is not a competitor of Gaussian, Inc. and that
 it will not use this program in any manner prohibited above.
 ---------------------------------------------------------------
  

 Cite this work as:
 Gaussian 16, Revision C.01,
 M. J. Frisch, G. W. Trucks, H. B. Schlegel, G. E. Scuseria, 
 M. A. Robb, J. R. Cheeseman, G. Scalmani, V. Barone, 
 G. A. Petersson, H. Nakatsuji, X. Li, M. Caricato, A. V. Marenich, 
 J. Bloino, B. G. Janesko, R. Gomperts, B. Mennucci, H. P. Hratchian, 
 J. V. Ortiz, A. F. Izmaylov, J. L. Sonnenberg, D. Williams-Young, 
 F. Ding, F. Lipparini, F. Egidi, J. Goings, B. Peng, A. Petrone, 
 T. Henderson, D. Ranasinghe, V. G. Zakrzewski, J. Gao, N. Rega, 
 G. Zheng, W. Liang, M. Hada, M. Ehara, K. Toyota, R. Fukuda, 
 J. Hasegawa, M. Ishida, T. Nakajima, Y. Honda, O. Kitao, H. Nakai, 
 T. Vreven, K. Throssell, J. A. Montgomery, Jr., J. E. Peralta, 
 F. Ogliaro, M. J. Bearpark, J. J. Heyd, E. N. Brothers, K. N. Kudin, 
 V. N. Staroverov, T. A. Keith, R. Kobayashi, J. Normand, 
 K. Raghavachari, A. P. Rendell, J. C. Burant, S. S. Iyengar, 
 J. Tomasi, M. Cossi, J. M. Millam, M. Klene, C. Adamo, R. Cammi, 
 J. W. Ochterski, R. L. Martin, K. Morokuma, O. Farkas, 
 J. B. Foresman, and D. J. Fox, Gaussian, Inc., Wallingford CT, 2019.
 
 ******************************************
 Gaussian 16:  ES64L-G16RevC.01  3-Jul-2019
                11-Nov-2020 
 ******************************************
 --------------------------------
 # b3lyp/6-311++g(d,p) output=wfn
 --------------------------------
 1/38=1,172=1/1;
 2/12=2,17=6,18=5,40=1/2;
 3/5=4,6=6,7=1111,11=2,25=1,30=1,74=-5/1,2,3;
 4//1;
 5/5=2,38=5/2;
 6/7=2,8=2,9=2,10=2,28=1/1;
 99/5=1,6=100,9=1/99;
 -------------------
 Single Point Energy
 -------------------
 Symbolic Z-matrix:
 Charge = -1 Multiplicity = 1
 C                    -1.56175   0.00024  -0.00077 
 H                    -1.22868  -0.8739   -0.54642 
 H                    -1.22899   0.90985  -0.48521 
 H                    -1.22821  -0.03505   1.02893 
 Br                    1.79745  -0.00002   0.0002 
 Cl                   -3.41435  -0.00006  -0.00011 
 
                          Input orientation:                          
 ---------------------------------------------------------------------
 Center     Atomic      Atomic             Coordinates (Angstroms)
 Number     Number       Type             X           Y           Z
 ---------------------------------------------------------------------
      1          6           0       -1.561746    0.000240   -0.000770
      2          1           0       -1.228676   -0.873898   -0.546424
      3          1           0       -1.228990    0.909846   -0.485210
      4          1           0       -1.228211   -0.035046    1.028934
      5         35           0        1.797449   -0.000023    0.000201
      6         17           0       -3.414353   -0.000057   -0.000113
 ---------------------------------------------------------------------
                    Distance matrix (angstroms):
                    1          2          3          4          5
     1  C    0.000000
     2  H    1.082955   0.000000
     3  H    1.082955   1.784794   0.000000
     4  H    1.082950   1.784776   1.784784   0.000000
     5  Br   3.359195   3.196856   3.197314   3.195956   0.000000
     6  Cl   1.852607   2.416451   2.416413   2.416481   5.211802
                    6
     6  Cl   0.000000
 This structure is nearly, but not quite of a higher symmetry.
 Consider Symm=Loose if the higher symmetry is desired.
 Stoichiometry    CH3BrCl(1-)
 Framework group  C1[X(CH3BrCl)]
 Deg. of freedom    12
 Full point group                 C1      NOp   1
 Largest Abelian subgroup         C1      NOp   1
 Largest concise Abelian subgroup C1      NOp   1
                         Standard orientation:                         
 ---------------------------------------------------------------------
 Center     Atomic      Atomic             Coordinates (Angstroms)
 Number     Number       Type             X           Y           Z
 ---------------------------------------------------------------------
      1          6           0        0.093594    0.346574    1.381613
      2          1           0        0.818379    0.941110    0.839390
      3          1           0        0.311385   -0.710165    1.288543
      4          1           0       -0.913718    0.565590    1.049748
      5         35           0       -0.125994   -0.469448   -1.869553
      6         17           0        0.213657    0.797335    3.174530
 ---------------------------------------------------------------------
 Rotational constants (GHZ):         157.4197465           0.7239671           0.7239670
 Standard basis: 6-311++G(d,p) (5D, 7F)
 There are   126 symmetry adapted cartesian basis functions of A   symmetry.
 There are   121 symmetry adapted basis functions of A   symmetry.
   121 basis functions,   210 primitive gaussians,   126 cartesian basis functions
    31 alpha electrons       31 beta electrons
       nuclear repulsion energy       160.8646930287 Hartrees.
 NAtoms=    6 NActive=    6 NUniq=    6 SFac= 1.00D+00 NAtFMM=   60 NAOKFM=F Big=F
 Integral buffers will be    131072 words long.
 Raffenetti 2 integral format.
 Two-electron integral symmetry is turned on.
 One-electron integrals computed using PRISM.
 NBasis=   121 RedAO= T EigKep=  2.86D-04  NBF=   121
 NBsUse=   121 1.00D-06 EigRej= -1.00D+00 NBFU=   121
 ExpMin= 3.50D-02 ExpMax= 4.40D+05 ExpMxC= 1.51D+04 IAcc=3 IRadAn=         5 AccDes= 0.00D+00
 Harris functional with IExCor=  402 and IRadAn=       5 diagonalized for initial guess.
 HarFok:  IExCor=  402 AccDes= 0.00D+00 IRadAn=         5 IDoV= 1 UseB2=F ITyADJ=14
 ICtDFT=  3500011 ScaDFX=  1.000000  1.000000  1.000000  1.000000
 FoFCou: FMM=F IPFlag=           0 FMFlag=      100000 FMFlg1=           0
         NFxFlg=           0 DoJE=T BraDBF=F KetDBF=T FulRan=T
         wScrn=  0.000000 ICntrl=       500 IOpCl=  0 I1Cent=   200000004 NGrid=           0
         NMat0=    1 NMatS0=      1 NMatT0=    0 NMatD0=    1 NMtDS0=    0 NMtDT0=    0
 Petite list used in FoFCou.
 Keep R1 ints in memory in canonical form, NReq=29147525.
 Requested convergence on RMS density matrix=1.00D-08 within 128 cycles.
 Requested convergence on MAX density matrix=1.00D-06.
 Requested convergence on             energy=1.00D-06.
 No special actions if energy rises.
 SCF Done:  E(RB3LYP) =  -3074.40333802     A.U. after   12 cycles
            NFock= 12  Conv=0.14D-08     -V/T= 2.0014

 **********************************************************************

            Population analysis using the SCF Density.

 **********************************************************************

 Orbital symmetries:
       Occupied  (A) (A) (A) (A) (A) (A) (A) (A) (A) (A) (A) (A)
                 (A) (A) (A) (A) (A) (A) (A) (A) (A) (A) (A) (A)
                 (A) (A) (A) (A) (A) (A) (A)
       Virtual   (A) (A) (A) (A) (A) (A) (A) (A) (A) (A) (A) (A)
                 (A) (A) (A) (A) (A) (A) (A) (A) (A) (A) (A) (A)
                 (A) (A) (A) (A) (A) (A) (A) (A) (A) (A) (A) (A)
                 (A) (A) (A) (A) (A) (A) (A) (A) (A) (A) (A) (A)
                 (A) (A) (A) (A) (A) (A) (A) (A) (A) (A) (A) (A)
                 (A) (A) (A) (A) (A) (A) (A) (A) (A) (A) (A) (A)
                 (A) (A) (A) (A) (A) (A) (A) (A) (A) (A) (A) (A)
                 (A) (A) (A) (A) (A) (A)
 The electronic state is 1-A.
 Alpha  occ. eigenvalues -- -482.55331-101.41914 -62.20624 -56.02406 -56.02398
 Alpha  occ. eigenvalues --  -56.02398 -10.09173  -9.33329  -8.41486  -7.09719
 Alpha  occ. eigenvalues --   -7.08845  -7.08845  -6.23792  -6.23759  -6.23759
 Alpha  occ. eigenvalues --   -2.35028  -2.35018  -2.35018  -2.34992  -2.34991
 Alpha  occ. eigenvalues --   -0.71868  -0.54976  -0.48623  -0.31464  -0.31463
 Alpha  occ. eigenvalues --   -0.28128  -0.18326  -0.18325  -0.05192  -0.05018
 Alpha  occ. eigenvalues --   -0.05018
 Alpha virt. eigenvalues --    0.10601   0.11737   0.14529   0.14530   0.17383
 Alpha virt. eigenvalues --    0.17898   0.19387   0.19387   0.20503   0.20506
 Alpha virt. eigenvalues --    0.21474   0.22097   0.22100   0.24643   0.27660
 Alpha virt. eigenvalues --    0.30147   0.30148   0.35826   0.40009   0.52908
 Alpha virt. eigenvalues --    0.52910   0.54762   0.58126   0.58126   0.60858
 Alpha virt. eigenvalues --    0.62054   0.62055   0.64831   0.75881   0.75881
 Alpha virt. eigenvalues --    0.79392   0.84128   0.88102   0.88102   0.89143
 Alpha virt. eigenvalues --    0.89147   0.94076   0.95773   0.95774   1.00233
 Alpha virt. eigenvalues --    1.00233   1.00874   1.03910   1.25339   1.25340
 Alpha virt. eigenvalues --    1.26062   1.56292   1.66827   1.66827   1.83955
 Alpha virt. eigenvalues --    1.83956   1.85402   2.10620   2.21843   2.21845
 Alpha virt. eigenvalues --    2.26245   2.26246   2.26456   2.44910   2.44910
 Alpha virt. eigenvalues --    2.53778   2.58172   2.63309   2.63311   2.65777
 Alpha virt. eigenvalues --    2.65779   3.06026   3.28554   3.86247   3.86248
 Alpha virt. eigenvalues --    4.39815   4.39818   4.40321   4.40324   4.47147
 Alpha virt. eigenvalues --    6.74951   7.80415   7.80416   7.84405   9.83308
 Alpha virt. eigenvalues --   24.01941  25.97840  25.97842  26.47691  48.06508
 Alpha virt. eigenvalues --  215.81857 290.01222 290.01223 290.050931020.81873
          Condensed to atoms (all electrons):
               1          2          3          4          5          6
     1  C    5.561692   0.398881   0.398893   0.398855  -0.037369  -0.074566
     2  H    0.398881   0.516945  -0.037392  -0.037375   0.004230  -0.038119
     3  H    0.398893  -0.037392   0.517010  -0.037380   0.004233  -0.038159
     4  H    0.398855  -0.037375  -0.037380   0.516820   0.004222  -0.038044
     5  Br  -0.037369   0.004230   0.004233   0.004222  35.923145   0.040165
     6  Cl  -0.074566  -0.038119  -0.038159  -0.038044   0.040165  17.142235
 Mulliken charges:
               1
     1  C   -0.646387
     2  H    0.192830
     3  H    0.192794
     4  H    0.192902
     5  Br  -0.938626
     6  Cl   0.006489
 Sum of Mulliken charges =  -1.00000
 Mulliken charges with hydrogens summed into heavy atoms:
               1
     1  C   -0.067862
     5  Br  -0.938626
     6  Cl   0.006489
 Electronic spatial extent (au):  <R**2>=           1311.1574
 Charge=             -1.0000 electrons
 Dipole moment (field-independent basis, Debye):
    X=              0.3396    Y=              1.2611    Z=              5.0248  Tot=              5.1918
 Quadrupole moment (field-independent basis, Debye-Ang):
   XX=            -44.6578   YY=            -46.4611   ZZ=            -75.3176
   XY=             -0.5208   XZ=             -2.0735   YZ=             -7.7353
 Traceless Quadrupole moment (field-independent basis, Debye-Ang):
   XX=             10.8210   YY=              9.0177   ZZ=            -19.8388
   XY=             -0.5208   XZ=             -2.0735   YZ=             -7.7353
 Octapole moment (field-independent basis, Debye-Ang**2):
  XXX=             -2.2429  YYY=             -7.1496  ZZZ=            -40.8716  XYY=             -0.1227
  XXY=             -1.4643  XXZ=             -8.5221  XZZ=             -1.5789  YZZ=             -6.1038
  YYZ=             -9.2051  XYZ=             -0.4388
 Hexadecapole moment (field-independent basis, Debye-Ang**3):
 XXXX=            -83.5636 YYYY=           -167.9895 ZZZZ=          -1650.5341 XXXY=            -12.0673
 XXXZ=            -48.5915 YYYX=            -11.8769 YYYZ=           -181.7440 ZZZX=            -57.8781
 ZZZY=           -216.3571 XXYY=            -41.3584 XXZZ=           -265.0570 YYZZ=           -287.2049
 XXYZ=            -59.2134 YYXZ=            -16.2064 ZZXY=             -6.8620
 N-N= 1.608646930287D+02 E-N=-7.674916397460D+03  KE= 3.069998261908D+03
 Unable to Open any file for archive entry.
 1\1\GINC-NODE531\SP\RB3LYP\6-311++G(d,p)\C1H3Br1Cl1(1-)\V69787FF\11-No
 v-2020\0\\# b3lyp/6-311++g(d,p) output=wfn\\Single Point Energy\\-1,1\
 C,0,-1.561746,0.00024,-0.00077\H,0,-1.228676,-0.873898,-0.546424\H,0,-
 1.22899,0.909846,-0.48521\H,0,-1.228211,-0.035046,1.028934\Br,0,1.7974
 49,-0.000023,0.000201\Cl,0,-3.414353,-0.000057,-0.000113\\Version=ES64
 L-G16RevC.01\State=1-A\HF=-3074.403338\RMSD=1.424e-09\Dipole=-2.042596
 5,0.0001929,-0.0006757\Quadrupole=-16.2978078,8.1488528,8.1489551,-0.0
 003276,-0.0010729,0.0000053\PG=C01 [X(C1H3Br1Cl1)]\\@
 The archive entry for this job was punched.

 Writing a WFN file to "frame.wfn"


 MARY HAD A LITTLE LAMB
      HIS FEET WERE BLACK AS SOOT,
           AND EVERYWHERE THAT MARY WENT
                HIS SOOTY FOOT HE PUT.

                    -- NONAME
 Job cpu time:       0 days  0 hours  0 minutes 27.8 seconds.
 Elapsed time:       0 days  0 hours  0 minutes 14.3 seconds.
 File lengths (MBytes):  RWF=     10 Int=      0 D2E=      0 Chk=      2 Scr=      1
 Normal termination of Gaussian 16 at Wed Nov 11 19:04:14 2020.
